<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDetailWidget</class>
 <widget class="QWidget" name="AccountDetailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>13</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>accountName</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="recentTransactionTableWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>325</y>
     <width>478</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 247, 249);</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>对方账户</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>交易金额</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>287</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>最近交易</string>
   </property>
  </widget>
  <widget class="QToolButton" name="moreTransactionBtn">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>282</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(24, 109, 205);</string>
   </property>
   <property name="text">
    <string>更多 + </string>
   </property>
  </widget>
  <widget class="QLabel" name="noTransactionLabel">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>344</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
color: rgb(178, 178, 178);</string>
   </property>
   <property name="text">
    <string>您最近没有交易</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>154</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>addr</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>154</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="copyBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>154</y>
     <width>11</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>15</y>
     <width>83</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>导出私钥   </string>
   </property>
  </widget>
  <widget class="QToolButton" name="transferBtn">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>185</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>    发起转账</string>
   </property>
  </widget>
  <widget class="QToolButton" name="upgradeBtn">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>15</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>升级</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>189</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0 </string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel2">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>189</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>余额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="identityLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
